src/platform/packages/private/kbn-grid-layout/grid/use_grid_layout_events/sensors/touch.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+export type UserTouchEvent = TouchEvent | React.TouchEvent<HTMLButtonElement>;
+
+export const isTouchEvent = (e: Event | React.UIEvent<HTMLElement>): e is UserTouchEvent => {
+  return 'touches' in e;
+};
+
+export const startTouchInteraction = ({
+  e,
+  onMove,
+  onEnd,
+  onStart,
+}: {
+  e: UserTouchEvent;
+  onStart: () => void;
+  onMove: (e: Event) => void;
+  onEnd: () => void;
+}) => {
+  if (e.touches.length > 1) return;
+
+  const handleEnd = () => {
+    e.target!.removeEventListener('touchmove', onMove);
+    onEnd();
+  };
+
+  e.target!.addEventListener('touchmove', onMove, { passive: false });
```

Comment:
`passive: false` is the default I think

Reply:
for touch events, `passive: true` is the default, but you are right to remove it and keep it as default. I had it before as `passive: false`, but I found a better way of handling the same behavior (which is scrolling on touchmove when not dragging the drag handle, but not when dragging - the `touch-action:none;` css property. Thanks for pointing it out!
989c3e7047db5348795f8e20178054cf73ac868c